connect/client.go

```diff
@@ -35,8 +36,8 @@ type Client interface {
 	GetItemsByTitle(title string, vaultUUID string) ([]onepassword.Item, error)
 	GetItemByTitle(title string, vaultUUID string) (*onepassword.Item, error)
 	CreateItem(item *onepassword.Item, vaultUUID string) (*onepassword.Item, error)
-	UpdateItem(item *onepassword.Item, vaultUUID string) (*onepassword.Item, error)
-	DeleteItem(item *onepassword.Item, vaultUUID string) error
+	UpdateItem(item *onepassword.Item) (*onepassword.Item, error)
+	DeleteItem(item *onepassword.Item) error
```

Comment:
This breaks all existing usage of these functions. You correctly identified that the `vaultUUID` is not being used, but can we handle this a bit more graceful for users that are using the existing function signature?

Reply:
Backwards compatibility completely slipped my mind, seemed like an easy improvement, when GoLand pointed out we don't use those parameters. Thanks for noticing!

Reply:
The fact that we have an argument that is not being used, could still be considered a bug. We should definitely document this. We could even make it a bit more robust by throwing an error in case of ambiguity, i.e. if one of these cases is true:
- The vault UUID of the item and the `vaultUUID` argument are an empty string.
- The vault UUID of the item and the `vaultUUID` argument are both non-empty strings but not equal to each other.

Does not have to be addressed in this MR, but may worth opening an issue for.